internal/graphapi/port_test.go

```diff
@@ -51,6 +52,8 @@ func TestFullLoadBalancerPortLifecycle(t *testing.T) {
 
 	// Delete the Port
 	deletedResp, err := graphTestClient().LoadBalancerPortDelete(ctx, createdPort.ID)
+	fmt.Println(err)
+	fmt.Println(deletedResp)
```

Comment:
```suggestion
	t.Logf("error: %+v", err)
	t.Logf("deletedResp: %+v", deletedResp)
```

Reply:
✅  Ah, forgot to remove those troubleshooting lines. Just went ahead and cleaned this up.